src/views/orderReport.tsx

```diff
@@ -0,0 +1,467 @@
+          "rgb(0, 227, 150, 0.6)",
+          "rgb(254, 176, 25, 0.6)"
+        ],
+        borderColor: "#161A1D",
+        borderWidth: 1,
+      },
+    ]
+    let chartData = { labels: ORDER_REPORT_CHART_LABELS.CHANNELWISE_ORDER_LABELS, datasets: datasets };
+    setChannelWiseOrderData(chartData);
+  }
+
+  const getPaymentWiseOrder = () => {
+    let data = hourlyReport.paymentWiseOrderReport;
+    let datasets = [
+      {
+        label: ORDER_PIE_CHART_LABELS.PAYMENTWISE_ORDER,
+        data: [data.pgOtherCC, data.pgPayPal, data.pgKlarna, data.pgGiftCard, data.pgOtherPlcc, data.pgApplePay, data.pgJCPDummy, data.pgCBVI, data.pgPLCC, data.pgPLCCT, data.pgInstaDummy, data.pgGoogleDummy, data.pgSephPay, data.pgStoreCredit],
+        backgroundColor: [
+          "rgb(0, 143, 251)",
+          "rgb(0, 227, 150)",
+          "rgb(254, 176, 25)",
+          "rgb(255, 69, 96)",
+          "rgb(119, 93, 208)",
+          "rgb(0, 143, 251)",
+          "rgb(0, 227, 150)",
+          "rgb(254, 176, 25)",
+          "rgb(255, 69, 96)",
+          "rgb(119, 93, 208)",
+          "rgb(0, 143, 251)",
+          "rgb(0, 143, 251)",
+          "rgb(0, 227, 150)",
+          "rgb(254, 176, 25)",
+          "rgb(255, 69, 96)",
+          "rgb(119, 93, 208)"
+        ],
+        hoverBackgroundColor: [
+          "rgb(0, 143, 251, 0.6)",
+          "rgb(0, 227, 150, 0.6)",
+          "rgb(254, 176, 25, 0.6)",
+          "rgb(255, 69, 96, 0.6)",
+          "rgb(119, 93, 208, 0.6)",
+          "rgb(0, 143, 251, 0.6)",
+          "rgb(0, 227, 150, 0.6)",
+          "rgb(254, 176, 25, 0.6)",
+          "rgb(255, 69, 96, 0.6)",
+          "rgb(119, 93, 208, 0.6)",
+          "rgb(0, 143, 251, 0.6)",
+          "rgb(0, 143, 251, 0.6)",
+          "rgb(0, 227, 150, 0.6)",
+          "rgb(254, 176, 25, 0.6)",
+          "rgb(255, 69, 96, 0.6)",
+          "rgb(119, 93, 208, 0.6)"
+        ],
+        borderColor: "#161A1D",
+        borderWidth: 1,
+      },
+    ]
+    let chartData = { labels: ORDER_REPORT_CHART_LABELS.PAYMENTWISE_ORDER_LABELS, datasets: datasets };
+    setPaymentWiseOrderData(chartData);
+  }
+
+  const getHourlyTrendData = () => {
+    let data = hourlyReport.hourlyHistoricalOrderData;
+    let todayData = {
+      label: HOURLY_TREND_DATA_LABELS.TODAY,
+      data: data.today,
+      borderColor: 'rgb(0, 143, 251)'
+    };
+
+    let yesterdayData = {
+      label: HOURLY_TREND_DATA_LABELS.YESTERDAY,
+      data: data.yesterday,
+      borderColor: 'rgb(0, 227, 150)'
+    };
+    let lastWeekData = {
+      label: HOURLY_TREND_DATA_LABELS.LAST_WEEK,
+      data: data.lastWeek,
+      borderColor: 'rgb(254, 176, 25)'
+    };
+    let lastMonthData = {
+      label: HOURLY_TREND_DATA_LABELS.LAST_MONTH,
+      data: data.lastMonth,
+      borderColor: 'rgb(255, 69, 96)'
+    };
+    let lastYearData = {
+      label: HOURLY_TREND_DATA_LABELS.LAST_YEAR,
+      data: data.lastYear,
+      borderColor: 'rgb(119, 93, 208)'
+    };
+
```

Comment:
Try this approach to reduce number of lines. 
In the current format, the code looks repetitive.
```
[
  [HOURLY_TREND_DATA_LABELS.TODAY,data.today,'rgb(0, 143, 251)'],
  [HOURLY_TREND_DATA_LABELS.YESTERDAY,data.yesterday,'rgb(0, 227, 150)'],
  [HOURLY_TREND_DATA_LABELS.LAST_WEEK,data.lastWeek,'rgb(254, 176, 25)'],
]
.map(({label, data, borderColor}) => ({
  label,
  data,
  borderColor,
}))
```

Reply:
done